### PR TITLE
Add Easyshortener to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**
New APP from the Easypanel Community

### CODE REVIEW

**Comment by Supernova3339** [comment]
An announcement in discord for this one would be kinda cool as I've been working on this for a little bit.

File: templates/easyshortener/index.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        `ASSET_URL=https://${input.domain}`,
+        `APP_ENV=production`,
+        `APP_DEBUG=false`,
+        `APP_KEY=base64:YzRhdzY1aWUxa3Q4bDJhbWJ6ajdycmUya3h5MzdrcmM=`,
+        ``,
+        `# Configuration`,
+        `EASYSHORTENER_ENABLE_REGISTRATION=true`,
+        `EASYSHORTENER_INSALLATION_ENV=easypanel`,
+        ``,
+        `# Database`,
```
Comment (Ponkhy) [inline templates/easyshortener/index.ts:25]:
```suggestion
```

File: templates/easyshortener/index.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      serviceName: input.appServiceName,
+      env: [
+        `APP_NAME=Easyshortener`,
+        `APP_URL=https://${input.domain}`,
+        `ASSET_URL=https://${input.domain}`,
+        `APP_ENV=production`,
+        `APP_DEBUG=false`,
+        `APP_KEY=base64:YzRhdzY1aWUxa3Q4bDJhbWJ6ajdycmUya3h5MzdrcmM=`,
+        ``,
+        `# Configuration`,
```
Comment (Ponkhy) [inline templates/easyshortener/index.ts:21]:
```suggestion
```

**Review by Ponkhy — CHANGES_REQUESTED** [review]
The following needs to be added in order to load the assets.
```js
domains: [
  {
    name: input.domain,
  },
],
```

File: templates/easyshortener/index.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `APP_NAME=Easyshortener`,
+        `APP_URL=https://${input.domain}`,
+        `ASSET_URL=https://${input.domain}`,
+        `APP_ENV=production`,
+        `APP_DEBUG=false`,
+        `APP_KEY=base64:YzRhdzY1aWUxa3Q4bDJhbWJ6ajdycmUya3h5MzdrcmM=`,
```
[the remark this answers is not in the document]
Comment (Supernova3339) [reply templates/easyshortener/index.ts:19]: There isn't a function for it yet

**Comment by Supernova3339** [comment]
> The following needs to be added in order to load the assets.
> ```js
> domains: [
>   {
>     name: input.domain,
>   },
> ],
> ```

I'll add that ASAP when I get home

**Review by Ponkhy — APPROVED** [review]
LGTM

**Pull request merged**